components/Layout.tsx

```diff
@@ -3,7 +3,7 @@ import AppNav from './AppNav'
 import Footer from './Footer'
 
 type Props = {
-  children: React.ReactElement
+  children: React.ReactElement | {}
```

Comment:
Can you remove this? We don't want Layout to be blank. Layout should always have children.